calyx-opt/src/passes/external_to_ref.rs

```diff
@@ -0,0 +1,53 @@
+use crate::traversal::{Action, ConstructVisitor, Named, VisResult, Visitor};
+use calyx_ir::{self as ir, GetAttributes, LibrarySignatures};
+use calyx_utils::CalyxResult;
+
+//Turns memory cell primitives with the `@external(1)` attribute into
+// `ref` memory cells without the `@external` attribute.
```

Comment:
Perhaps you want this to be a Rust documentation comment. Those are indicated with triple-slashes, `///`.